Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -40,15 +40,17 @@ public partial class ActUIElementEditPage : Page
     {
         ActUIElement mAction;
         PlatformInfoBase mPlatform;
-        
+
         public ActUIElementEditPage(ActUIElement act)
         {
             InitializeComponent();
             mAction = act;
+            mAction.PropertyChanged += Action_PropertyChanged;
             ePlatformType ActivityPlatform = GetActionPlatform();
             mPlatform = PlatformInfoBase.GetPlatformImpl(ActivityPlatform);
             ElementTypeComboBox.BindControl(mAction, ActUIElement.Fields.ElementType, mPlatform.GetPlatformUIElementsType());
             ElementLocateByComboBox.BindControl(mAction, ActUIElement.Fields.ElementLocateBy, mPlatform.GetPlatformUIElementLocatorsList());
+            ElementLocateByComboBox.SelectedValue = Enum.GetName(typeof(eLocateBy), eLocateBy.POMElement);
             ShowPlatformSpecificPage();
```

Comment:
you disabeled the POM ENUM in case of not activate Beta flag so it won't demage             ElementLocateByComboBox.SelectedValue defualt value setting ?
 ElementLocateByComboBox.SelectedValue = Enum.GetName(typeof(eLocateBy), eLocateBy.POMElement);